lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/form/SearchField.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Untabable TextField as SearchField for VirtualSelectBox
+ */
+
```

Comment:
ok for callbackery, but for the qx implementation this seems odd ... haveing a special widget only to be used in a single context